bigquery/indexer.py

```diff
@@ -240,13 +243,56 @@ def read_table(client, table_name):
         client.dataset(dataset_id, project=project_id).table(table_name))
 
 
+def create_samples_json_export_file(es, index_name, project_id):
+    """
+    Writes the samples export JSON file to a GCS bucket. This significantly 
+    speeds up exporting the samples table to Saturn in the Data Explorer.
+
+    Args:
+        es: Elasticsearch object.
+        index_name: Name of Elasticsearch index.
+        project_id: Google Cloud Project ID containing the export samples bucket
+    """
+    entities = []
+    search = Search(using=es, index=index_name)
+    for hit in search.scan():
+        participant_id = hit.meta['id']
+        doc = hit.to_dict()
+        for sample in doc.get('samples', []):
+            sample_id = sample['sample_id']
+            export_sample = {'participant': participant_id}
+            for table_column, value in sample.iteritems():
+                # Ignore _has_* and sample_id fields.
+                splits = table_column.split('.')
+                if len(splits) != 4:
+                    continue
+                export_sample[splits[3]] = value
+
+            entities.append({
+                'entityType': 'sample',
+                'name': sample_id,
+                'attributes': export_sample,
+            })
+
+    client = storage.Client(project=project_id)
+    bucket_name = '%s-export-samples' % project_id
+    try:
+        bucket = client.get_bucket(bucket_name)
```

Comment:
Use [lookup_bucket](https://googleapis.github.io/google-cloud-python/latest/storage/client.html#google.cloud.storage.client.Client.lookup_bucket) instead, then you don't need to catch an exception

Reply:
Done.